images/iscsi-demo-target-tgtd/Dockerfile

```diff
@@ -16,7 +16,9 @@ RUN curl \
 # Add cirros
 RUN curl \
       http://download.cirros-cloud.net/0.3.5/cirros-0.3.5-x86_64-disk.img \
-      > /volume/cirros.img
```

Comment:
We can do that in a single command:

`qemu-img convert -p http://download.cirros-cloud.net/0.3.5/cirros-0.3.5-x86_64-disk.img -O raw /volume/cirros.img`